varcode/variant.py

```diff
@@ -248,18 +251,18 @@ def __eq__(self, other):
             self.ensembl == other.ensembl)
 
     def __getstate__(self):
-        fields = self.fields()
-        return {
-            field: fields[i] for (i, field) in enumerate(fields._fields)
-        }
+        # When pickle.load'ing, we want the original values vs. the normalized values.
+        # Normalization will happen upon object creation.
+        return [self.contig, self.original_start, self.original_ref, self.original_alt, self.ensembl.release]
 
     def __setstate__(self, fields):
-        # This field require special logic.
-        self.ensembl = cached_release(fields.pop("release"))
+        # Special logic needed for JSON serialization, since EnsemblRelease
+        # is not JSON serializable.
+        # TODO: this does not work for Genome objects; only EnsemblRelease.
```

Comment:
Not sure the best way to do that. I imagine we'd need to add JSON serialization for a lot of different classes. (Pickling already works with `Genome` objects.)

I figured I'd limit this PR to fixing one clear bug: original values weren't being reconstituted.